qa/rustfmt.sh

```diff
@@ -0,0 +1,10 @@
+r=0
```

Comment:
Lets please make this a proper script.. Start with `#! /bin/sh`, little comment what it does.

Reply:
ok